### PR TITLE
CLDR-13901 Add Kaingang locale to CLDR

### PR DESCRIPTION
**Pull request by robertgmelo**
##### Checklist

- [X] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-13901
- [X] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

**Comment by srl295** [comment]
I recommend to retarget this at CLDR-13568 which is already accepted.

File: common/supplemental/coverageLevels.xml [hunk 1]
```diff
@@ -433,11 +433,11 @@ For terms of use, see http://www.unicode.org/copyright.html
 		<coverageLevel value="basic" match="listPatterns/listPattern/listPatternPart[@type='(2|start|middle|end)']"/>
 		<coverageLevel value="basic" match="units/durationUnit[@type='(hm|ms|hms)']/durationUnitPattern"/>
 		<coverageLevel value="moderate" match="localeDisplayNames/languages/language[@type='%language60']"/>
-		
+
```
Comment (srl295) [inline common/supplemental/coverageLevels.xml:436]: spurious changes, probably tabs

Comment (robertgmelo) [reply common/supplemental/coverageLevels.xml:436]: should I bring back those tabs? Or I can leave them removed, as the rest of the file. I think we can go ahead.

File: common/main/kgp.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (macchiati) [inline common/main/kgp.xml:1]: Looks good, except that we put locales in /seed/ until they get fleshed out.

Comment (robertgmelo) [reply common/main/kgp.xml]: Noted. I've updated the PR. Tks, Mark.

File: common/supplemental/coverageLevels.xml (hunk 1 above)
Comment (srl295) [reply common/supplemental/coverageLevels.xml:436]: leave them

**Comment by srl295** [comment]
I opened a PR into your PR ^ 

I get this error message. I'm not sure how to fix it though. @macchiati or others do you know what is missing?

```
     [java] localesForNames.containsAll(coverageLocales) Missing:	1
     [java] 	kgp	Kaingang
     [java]   TestLanguageNameCoverage {
     [java]     Error: (TestCLDRLocaleCoverage.java:96) : localesForNames.containsAll(coverageLocales): expected true
```

**Comment by robertgmelo** [comment]
@srl295 this is the error I've mentioned to you when we discussed on Slack. Indeed, it also appeared to me when building CLDR with those changes and after some attempts I didn't find a solution.

I've put kgp in the coverage level _(%language80)_ assuming this could be the lowest, not sure if it's reason. But after Mark guided me to move to **seed**, I'm wondering now if the coverage level should be _%languagecomp_, what do you think?

I also suspected that **kgp** should be in English data to guarantee a probable fallback, but **kgp** is already there.

**Comment by srl295** [comment]
@robertgmelo I'm really not sure. I don't recognize the error message. Whatever it is, it's clearly not documented well enough in requirements fore new locales.

Can you try the change you proposed and see if it makes a difference?

**Comment by robertgmelo** [comment]
@srl295 Yes, if I remove **kgp** from _coverageLevels.xml_, the check works without errors.

After dig into `TestCLDRLocaleCoverage`, I could understand the _error_:
- It happens because the test retrieves the locale names from `CLDRConfig.getCldrFactory()` (which considers only _common dir, not _seed_) and combines with a hard-coded list of items. 
- Since **kgp** is now on _seed_, its locale name is not found.

Not sure if it's the real intention to check only on _common_, but Mark can confirm. In case we also should consider both _main_ and _seed_, we could use `CLDRConfig.getFullCldrFactory()`. Is that intentional @macchiati ? Probably consider both **common** and **seed** is good, so we would check if core data for new locales are being added properly. 

Maybe the right thing is **do not add locale content on _coverageLevels.xml_**. I can take further actions as soon Mark provide his thoughts.

**Comment by srl295** [comment]
Sounds like a bug- please file a new bug and remove from coverage levels so
your PR can move forward. You can reject my PR and make it part of the new
bug.

**Comment by robertgmelo** [comment]
@srl295 changes applied.

New issue created: CLDR-13993. I'll make the changes accordingly and submit a separated PR.

**Comment by srl295** [comment]
@macchiati ok to merge?

File: seed/main/kgp.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/ for derived annotations.
+-->
+<ldml>
+	<identity>
+		<version number="$Revision$"/>
+		<language type="kgp"/>
+	</identity>
+	<localeDisplayNames>
+		<languages>
+			<language type="kgp">Kanhgág</language>
```
Comment (Liggliluff) [inline seed/main/kgp.xml:19]: I just want to make sure, do you have a source on that you should write language names with an initial-uppercased letter in this language? A common mistake when looking at the English file as reference is to think that language names should be in initial-uppercase in all translations (as well for months and weekdays). Wikipedia has the native language name written fully in lowercase: [Kaingang language](https://en.wikipedia.org/wiki/Kaingang_language), so either this file or the article needs updating.

Comment (robertgmelo) [reply seed/main/kgp.xml:19]: Once you post the comment here, I've sent the question to Kaingang linguists. Today I've received a feedback, and it was explained to me it was a practice in the past, but not rule, where they've used capital letter for languages as a way of show importance/relevance for the language. In addition, he confirmed the rule should be the same as Portuguese (lowercase). I'll update the PR right now.

**Comment by srl295** [comment]
@robertgmelo please rebase and it might pass

**Comment by robertgmelo** [comment]
Thanks, Mark and Steven. I'll be working on that in the next months.

Please feel free to merge, as I don't have merge permissions on cldr repo.

**Pull request merged**